doc/man3/SSL_CTX_load_verify_locations.pod

```diff
@@ -73,41 +84,43 @@ format. The file can contain several CA certificates identified by
 sequences. Before, between, and after the certificates text is allowed
 which can be used e.g. for descriptions of the certificates.
 
-The B<CAfile> is processed on execution of the SSL_CTX_load_verify_locations()
+The I<CAfile> is processed on execution of the SSL_CTX_load_verify_locations()
 function.
 
-If B<CApath> is not NULL, it points to a directory containing CA certificates
-in PEM format. The files each contain one CA certificate. The files are
+If I<CApath> is not NULL, it points to a directory containing CA certificates
+in PEM format. The files each contain exactly one CA certificate. The files are
 looked up by the CA subject name hash value, which must hence be available.
-If more than one CA certificate with the same name hash value exist, the
+If more than one CA certificate with the same subject name hash value exist, the
 extension must be different (e.g. 9d66eef0.0, 9d66eef0.1 etc). The search
 is performed in the ordering of the extension number, regardless of other
 properties of the certificates.
-Use the B<c_rehash> utility to create the necessary links.
+Use the L<openssl-rehash(1)> utility to create the necessary links.
 
-The certificates in B<CApath> are only looked up when required, e.g. when
-building the certificate chain or when actually performing the verification
-of a peer certificate.
+The certificate files in I<CApath> are only looked up when required, e.g., when
+building the certificate chain of the own client or server certificate
+or when actually performing the verification of a peer certificate.
 
 When looking up CA certificates for chain building, the OpenSSL library
-will search for suitable certificates first in B<CAfile>, then in B<CApath>.
+will search for suitable certificates first in I<CAfile>, then in I<CApath>.
 Details of the chain building process are described in
-L<openssl-verification-options(1)/Certification Path Building>.
+L<openssl-verification-options(1)/Certification Path< Building>.
 
-If B<CAstore> is not NULL, it's a URI for to a store, which may
+I<CAstore> must not be NULL. It must be a URI to a store, which may
 represent a single container or a whole catalogue of containers.
-Apart from the B<CAstore> not necessarily being a local file or
-directory, it's generally treated the same way as a B<CApath>.
+If I<CAstore> starts with the scheme C<file:>, it is treated like a local file
+or directory. Apart from the I<CAstore> not necessarily being a local file or
+directory, it's generally treated the same way as a I<CApath>
+if the store behaves like a directory, otherwise like I<CAfile>.
```

Comment:
```suggestion
or directory.
```

Reply:
Why don't you like this extra information?
I just added `if the store behaves like a directory, otherwise like I<CAfile>`, which should be correct.